common/script_values/wc_mana_values.txt

```diff
@@ -0,0 +1,191 @@
+﻿wc_sorcerer_trait_1_mana_value = 10
+wc_sorcerer_trait_noninheritable_mana_value = 15
+wc_sorcerer_trait_2_mana_value = 20
+wc_sorcerer_trait_3_mana_value = 30
+
+wc_mana_per_perk_base_value = 1
+
+wc_max_mana_value = {
+	add = { # Light
+		add = light_magic_lifestyle_perks
+		add = light_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Shadow
+		add = shadow_magic_lifestyle_perks
+		add = shadow_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Disorder
+		add = disorder_magic_lifestyle_perks
+		add = disorder_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Order
+		add = order_magic_lifestyle_perks
+		add = order_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Life
+		add = life_magic_lifestyle_perks
+		add = life_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Death
+		add = death_magic_lifestyle_perks
+		add = death_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	# Elemental
+	add = { # Fire
+		add = elemental_fire_magic_lifestyle_perks
+		add = elemental_fire_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Water
+		add = elemental_water_magic_lifestyle_perks
+		add = elemental_water_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Air
+		add = elemental_air_magic_lifestyle_perks
+		add = elemental_air_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+	add = { # Death
+		add = elemental_earth_magic_lifestyle_perks
+		add = elemental_earth_magic_lifestyle_perk_points
+		multiply = wc_mana_per_perk_base_value
+	}
+
+	if = {
+		limit = { has_trait = magic_good_1 }
+		add = wc_sorcerer_trait_1_mana_value
+	}
+	else_if = {
+		limit = { has_trait = magic_good_2 }
+		add = wc_sorcerer_trait_2_mana_value
+	}
+	else_if = {
+		limit = { has_trait = magic_good_3 }
+		add = wc_sorcerer_trait_3_mana_value
+	}
+	else_if = {
+		limit = { has_trait = magic_bad_1 }
+		subtract = wc_sorcerer_trait_1_mana_value
+	}
+	else_if = {
+		limit = { has_trait = magic_bad_2 }
+		subtract = wc_sorcerer_trait_2_mana_value
+	}
+	else_if = {
+		limit = { has_trait = magic_bad_3 }
+		subtract = wc_sorcerer_trait_3_mana_value
+	}
+
+	if = {
+		limit = { has_trait = magic_good_noninheritable }
+		add = wc_sorcerer_trait_noninheritable_mana_value
+	}
+	else_if = {
+		limit = { has_trait = magic_bad_noninheritable }
+		subtract = wc_sorcerer_trait_noninheritable_mana_value
+	}
+
+	ceiling = yes
```

Comment:
You subtract so to avoid it going negative.
```suggestion
	min = 0
	ceiling = yes
```

Reply:
done